docs/Additional-Reading.md

```diff
@@ -38,11 +38,11 @@
 * [MSDN Rx forum](http://social.msdn.microsoft.com/Forums/en-US/home?forum=rx)
 
 # RxJS
-* [the RxJS github site](http://reactive-extensions.github.io/RxJS/)
+* [the RxJS github site](https://github.com/Reactive-Extensions/RxJS)
```

Comment:
It would be better to link to the most recent site: `https://github.com/reactivex/rxjs`